src/screens/Menu/Menu.js

```diff
@@ -252,11 +253,19 @@ const Menu = ({
               </FooterBanner>
             )}
             <LinksSection>
-              <LegalTextLink onPress={() => openLegalModal(ENDPOINTS.TERMS_OF_SERVICE)}>
+              <LegalTextLink
+                onPress={() =>
+                  openLegalModal(firebaseRemoteConfig.getString(REMOTE_CONFIG.PRISMIC_TERMS_OF_POLICY_DOCUMENT_ID))
```

Comment:
To make it easier to read maybe you can put `firebaseRemoteConfig.getString(REMOTE_CONFIG.PRISMIC_TERMS_OF_POLICY_DOCUMENT_ID)` as variable and have one liner like `onPress={() => openLegalModal(prismicTermsOfPolicyDocumentId)}`. Same for `PRISMIC_PRIVACY_POLICY_DOCUMENT_ID` below, wdyt?